loki/transformations/tests/test_block_index_inject.py

```diff
@@ -0,0 +1,386 @@
+# (C) Copyright 2018- ECMWF.
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+# In applying this licence, ECMWF does not waive the privileges and immunities
+# granted to it by virtue of its status as an intergovernmental organisation
+# nor does it submit to any jurisdiction.
+
+from shutil import rmtree
+import pytest
+
+from loki import (
+    Dimension, gettempdir, Scheduler, OMNI, FindNodes, Assignment, FindVariables, CallStatement, Subroutine,
+    Item, available_frontends
+)
+from loki.transformations import BlockViewToFieldViewTransformation, InjectBlockIndexTransformation
+
+@pytest.fixture(scope='module', name='horizontal')
+def fixture_horizontal():
+    return Dimension(name='horizontal', size='nlon', index='jl', bounds=('start', 'end'),
+                     aliases=('nproma',), bounds_aliases=('bnds%start', 'bnds%end'))
+
+
+@pytest.fixture(scope='module', name='blocking')
+def fixture_blocking():
+    return Dimension(name='blocking', size='nb', index='ibl', index_aliases='bnds%kbl')
+
+
+@pytest.fixture(scope='module', name='config')
```

Comment:
I recommend making this a function-scoped fixture, which allows you to edit the config dict inside the test without having to make a copy first.